frontend/src/data/variables/AcsPovertyProvider.ts

```diff
@@ -50,6 +57,11 @@ class AcsPovertyProvider extends VariableProvider {
     // Calculate totals where dataset doesn't provide it
     // TODO- this should be removed when Totals come from the Data Server
     const calculatedValueForAll = df
+      .where(
```

Comment:
Ditto for adding a comment here about why these are removed

Also, did you mean to exclude HISPANIC rather than TWO_OR_MORE?

Reply:
Done